README.md

```diff
@@ -14,257 +14,32 @@ pgvecto.rs is a Postgres extension that provides vector similarity search functi
-
-Options for table `flat`.
-
-| Key          | Type  | Description                                |
-| ------------ | ----- | ------------------------------------------ |
-| quantization | table | The algorithm to be used for quantization. |
-
-Options for table `ivf`.
-
-| Key              | Type    | Description                                                     |
-| ---------------- | ------- | --------------------------------------------------------------- |
-| nlist            | integer | Number of cluster units. Default value is `1000`.               |
-| nprobe           | integer | Number of units to query. Default value is `10`.                |
-| least_iterations | integer | Least iterations for K-Means clustering. Default value is `16`. |
-| iterations       | integer | Max iterations for K-Means clustering. Default value is `500`.  |
-| quantization     | table   | The quantization algorithm to be used.                          |
-
-Options for table `hnsw`.
-
-| Key             | Type    | Description                                        |
-| --------------- | ------- | -------------------------------------------------- |
-| m               | integer | Maximum degree of the node. Default value is `12`. |
-| ef_construction | integer | Search scope in building.  Default value is `300`. |
-| quantization    | table   | The quantization algorithm to be used.             |
-
-Options for table `quantization`.
-
-| Key     | Type  | Description                                         |
-| ------- | ----- | --------------------------------------------------- |
-| trivial | table | If this table is set, no quantization is used.      |
-| scalar  | table | If this table is set, scalar quantization is used.  |
-| product | table | If this table is set, product quantization is used. |
-
-You can choose only one algorithm in above indexing algorithms. Default value is `trivial`.
-
-Options for table `product`.
-
-| Key    | Type    | Description                                                                                                              |
-| ------ | ------- | ------------------------------------------------------------------------------------------------------------------------ |
-| sample | integer | Samples to be used for quantization. Default value is `65535`.                                                           |
-| ratio  | string  | Compression ratio for quantization. Only `"x4"`, `"x8"`, `"x16"`, `"x32"`, `"x64"` are allowed. Default value is `"x4"`. |
-
-And you can change the number of expected result (such as `ef_search` in hnsw) by using the following SQL.
-
-```sql
----  (Optional) Expected number of candidates returned by index
-SET vectors.k = 32;
---- Or use local to set the value for the current session
-SET LOCAL vectors.k = 32;
+```sh
+docker run --name pgvecto-rs-demo -e POSTGRES_PASSWORD=mysecretpassword -p 5432:5432 -d tensorchord/pgvecto-rs:pg16-latest
```

Comment:
We dont have such a tag.

https://hub.docker.com/repository/docker/tensorchord/pgvecto-rs/tags?page=1&ordering=last_updated&name=pg16-latest

Reply:
it should be fixed in #159